llvm/test/MC/Xtensa/Core/invalid.s

```diff
@@ -4,10 +4,6 @@ LBL0:
 
 # Out of range immediates
 
-# imm12m
-movi a1, 3000
```

Comment:
Lost the range verification test?

Reply:
I removed this test, because constant pool is implemented now. The Xtensa asm allows mnemonics like "movi a1, imm" even if "imm" is out of range, we can just replace it with loading contant from constant pool